### PR TITLE
Add web support for Image Picker

### PR DESCRIPTION
**Pull request by vin-fandemand**
I took the liberty of adding web support for Image Picker (most of the other widgets work with web even though you haven't officially declared it). The main difference here is that `File` object does not exist in web. Hence the data from the picked Image is passed as `Uint8List`, which is then displayed as `Image` class (as we do for other types of images). 
Before my changes, the image picked by Image Picker was passed back as `Image` object. This would create problems as once it is an `Image` object, there is no way the developer can use the raw data (for storing, further manipulation etc). Hence I used `Uint8List` instead. Developers can then handle the data type based on whether they use the object in web or mobile.

### CODE REVIEW

File: lib/src/fields/form_builder_image_picker.dart [hunk 1]
```diff
@@ -158,9 +159,11 @@ class _FormBuilderImagePickerState extends State<FormBuilderImagePicker> {
                             width: widget.imageWidth,
                             height: widget.imageHeight,
                             margin: widget.imageMargin,
-                            child: item is String
-                                ? Image.network(item, fit: BoxFit.cover)
-                                : Image.file(item, fit: BoxFit.cover),
+                            child: kIsWeb
+                                ? Image.memory(item, fit: BoxFit.cover)
```
Comment (vin-fandemand) [inline lib/src/fields/form_builder_image_picker.dart:163]: As the data is passed back as `Uint8List`

File: lib/src/widgets/image_source_sheet.dart [hunk 2]
```diff
@@ -25,7 +26,7 @@ class ImageSourceBottomSheet extends StatelessWidget {
   /// Callback when an image is selected.
   ///
   /// **Note**: This will work on web platform whereas [onImageSelected] will not.
-  final Function(Image) onImage;
+  final Function(Uint8List) onImage;
```
Comment (vin-fandemand) [inline lib/src/widgets/image_source_sheet.dart:29]: `Uint8List` give developers more options to manipulate / store etc. Image is only useful for display purposes.

File: lib/src/widgets/image_source_sheet.dart [hunk 3]
```diff
@@ -65,19 +66,18 @@ class ImageSourceBottomSheet extends StatelessWidget {
...
-
-      if (null != onImageSelected) {
-        // Warning:  this will not work on the web platform because pickedFile
-        // will instead point to a network resource.
-        assert(!kIsWeb);
-
-        final imageFile = File(pickedFile.path);
-        assert(null != imageFile);
-        onImageSelected(imageFile);
+      if (kIsWeb) {
```
Comment (vin-fandemand) [inline lib/src/widgets/image_source_sheet.dart:69]: I removed the assertion that it should only work for mobile and made it into an if-else.

**Comment by danvick** [comment]
LGTM.

Thanks for your contribution @vin-fandemand. You're the best

**Pull request merged**